File: tests/fixtures/go/slop_elision_pt_br.go
package main

import "net/http"

func HandleRequest(w http.ResponseWriter, r *http.Request) {
	checkAuth(r)
	// ... resto do código sem alteração // expect: SLOP001
	w.WriteHeader(200)
}

func checkAuth(r *http.Request) {
	_ = r
}

File: tests/fixtures/rust/slop_elision_pt_br.rs
fn processar_dados(entrada: Vec<u8>) -> Result<String, ()> {
    validar(&entrada)?;
    // ... resto do código sem alteração // expect: SLOP001
    Ok(String::new())
}

fn validar(_entrada: &[u8]) -> Result<(), ()> {
    Ok(())
}
